src/main/resources/com/google/api/codegen/java/sample.snip

```diff
@@ -64,6 +62,9 @@
       @end
 
       {@class.requestBodyTypeName} {@class.requestBodyVarName} = new {@class.requestBodyTypeName}();
+      @join field : class.requestBodyFields if class.requestBodyFields
```

Comment:
I actually added the `@join ... if` syntax to snippet language, it's semantically equivalent to
```
@if ...
  @join ...
  @end
@end
```

Unfortunately... in certain contexts the whitespace rules break down, so I fully expand the loops in those cases. I prefer where it works to use the short `@join if` syntax